app.py

```diff
@@ -86,12 +106,95 @@ def delete(id):
         return redirect(url_for('history'))
 
 
+#Database API for getting charts
+@app.route('/request/chart/<id>', methods=['POST', 'GET'])
+def updateChart(id):
+    getDate = records.query.order_by()
+    toAppend = []
+    for n in getDate:
+        toAppend.append(n.time)
+    sortedArray = sorted(toAppend, key=sortingDates)
+    #function to add up all records by id
```

Comment:
Hier würde es evtl. Sinn machen, das in eine separate Funktion zu packen und entsprechend zu benennen.